.github/workflows/builds.yml

```diff
@@ -46,6 +46,9 @@ jobs:
                  -DVECMEM_BUILD_BENCHMARKING=TRUE
                  -S ${{ github.workspace }} -B build
                  ${{ matrix.PLATFORM.GENERATOR }}
+    # Print the configuration log for debugging
+    - name: Print CMake Configure Log
+      run: cat build/CMakeFiles/CMakeOutput.log
```

Comment:
`CMakeOutput.log`, as it turns out, only contains the successful compilation attempts. The failed ones go into `CMakeError.log`. So it would be even more important to print the content of that one. (Printing them one after the other should be fine.)